.github/workflows/cron-reservationExpiry.yml

```
@@ -0,0 +1,23 @@
+name: Cron - reservationExpiry
+
+on:
+  # "Scheduled workflows run on the latest commit on the default or base branch."
+  # — https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
+  schedule:
+    # Runs "At every minute." (see https://crontab.guru)
+    - cron: "* * * * *"
```

actionlint (1.7.7)

scheduled job runs too frequently. it runs once per 60 seconds. the shortest interval is once every 5 minutes

(events)